roles/pulp_database_config/tasks/main.yml

```diff
@@ -20,23 +20,28 @@
 - meta: flush_handlers
 
 - block:
-    - name: Run database auth migrations
-      command: '{{ pulp_django_admin_path }} migrate auth --no-input'
-      register: migrate_auth
-      changed_when: "'No migrations to apply' not in migrate_auth.stdout"
-      environment:
-        # Stops django-guardian from creating an anonymous user
-        PULP_ANONYMOUS_USER_NAME: '@none None'
-
     - name: Run database migrations
       command: '{{ pulp_django_admin_path }} migrate --no-input'
       register: result
       changed_when: "'No migrations to apply' not in result.stdout"
+      environment:
+        # Stops django-guardian from creating an anonymous user
+        PULP_ANONYMOUS_USER_NAME: '@none None'
```

Comment:
- [ ] Check with galaxy if this is still needed.

Reply:
@mdellweg bmbouter and I had a meeting, we want to retain creating the django-guardian anonymous user. partially for consistency, it has already been created on 3.6.0 through 3.6.3 installs. So these 3 lines should be removed.